ui/packages/catalog-ui-search/src/main/webapp/js/controllers/cesium.layer-ordering.spec.js

```diff
@@ -118,6 +118,7 @@ describe('Cesium Layer Ordering', () => {
       })
       describe('Not all layers initialized', () => {
         const prev = ['b', 'c', 'e']
+        // eslint-disable-next-line no-undef
```

Comment:
why would the next lline need this comment?

Reply:
good point - looks like we need to update our eslint config 
> By the same token, supporting ES6 syntax is not the same as supporting new ES6 globals (e.g., new types such as Set). For ES6 syntax, use { "parserOptions": { "ecmaVersion": 6 } }; for new ES6 global variables, use { "env": { "es6": true } }

https://eslint.org/docs/user-guide/configuring#specifying-parser-options

Reply:
added